README.md

```diff
@@ -79,6 +79,35 @@ x1 shifted circularly by -1 = [0.5, 0.75, 1.0, 2.0, 3.0, 4.0, 5.0, 0.25]
 x1 shifted circularly by 2 = [4.0, 5.0, 0.25, 0.5, 0.75, 1.0, 2.0, 3.0]
 ```
 
+### Memory issues
```

Comment:
Please update this section with the updates from https://github.com/hpsc-lab/OpenFHE.jl/pull/64 once we have merged the other PR